google-cloud-spanner/acceptance/spanner/client/single_use_test.rb

```diff
@@ -161,13 +161,38 @@
   end
 
   def assert_accounts_equal expected, actual
-    expected[:account_id].must_equal actual[:account_id]
-    expected[:username].must_equal actual[:username]
-    expected[:reputation].must_equal actual[:reputation]
-    expected[:active].must_equal actual[:active]
+    if actual[:account_id].nil?
```

Comment:
I'm curious why we need to do this. Is the semantic difference significant? (Is the gapic overriding `.nil?`?)

Reply:
Minitest warns if you call `must_equal nil`. The next version of Minitest will switch to raising an error if you call that. So if when these values are `nil`, which is perfectly valid for them to be, we get warnings. So this change was simply to remove the warnings.